Makefile

```
@@ -0,0 +1,32 @@
+# Colors for pretty output
+BLUE := \033[36m
+BOLD := \033[1m
+GREEN := \033[32m
+RESET := \033[0m
+
+.DEFAULT_GOAL := help
+
+.PHONY: build clean book check
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)